rpcd/playbooks/roles/rpc_maas/tasks/agent_install_remote.yml

```diff
@@ -13,22 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Verify raxmon-entities-list and MaaS API authentication does not fail
-  shell: |
```

Comment:
Are you removing this task because it runs against a large number of hosts so increases the chance of failure?